randovania/games/prime2/generator/hint_distributor.py

```diff
@@ -36,8 +36,13 @@ def num_joke_hints(self) -> int:
         return 2
 
     @override
-    def less_interesting_pickup_to_hint(self, pickup: PickupEntry) -> bool:
-        return not pickup.has_hint_feature("key")
+    def hint_suitability_for_pickup(self, pickup: PickupEntry) -> HintSuitability:
+        if pickup.show_in_credits_spoiler:
+            return HintSuitability.INTERESTING
+        elif not pickup.has_hint_feature("key"):
+            return HintSuitability.LESS_INTERESTING
```

Comment:
you've introduced a bug here. keys are shown in the credits spoiler, so now they'll always be considered INTERESTING. that's why I designed these functions the way I did, so that interesting was a strict subset of less interesting